### ISSUE TITLE
Training model for video data

### ISSUE THREAD

**Issue by bhargavaurala** [MhLiao/TextBoxes#90]
Hello,

I am trying to finetune TextBoxes ICDAR model for a video dataset. Because it is a video I cannot use lmdb as the size would be too large. So I am planning to pass every frame to a caffe net object in python using code similar to examples/TextBoxes/demo.py and then using solver.step(1) to perform training.

Could you please let me know the correct preprocessing and format to pass the ground truth annotations  of text bounding boxes (I have annotations per frame) to the data layer so that we can compute multibox loss?

Please note I cannot use the create_dataset.sh because I have video dataset and not a list of images with corresponding ground truth files.

Bhargava
